contracts/BaseStrategy.sol

```diff
@@ -432,23 +432,14 @@ abstract contract BaseStrategy {
     function adjustPosition(uint256 _debtOutstanding) internal virtual;
 
     /**
-     * Make as much capital as possible "free" for the Vault to take. Some
-     * slippage is allowed, since when this method is called the strategist is
-     * no longer receiving their performance fee. The goal is for the Strategy
-     * to divest as quickly as possible while not suffering exorbitant losses.
-     * This function is used during emergency exit instead of
-     * `prepareReturn()`. This method returns any realized losses incurred,
-     * and should also return the amount of `want` tokens available to repay
-     * outstanding debt to the Vault.
-     */
-    function exitPosition(uint256 _debtOutstanding)
-        internal
-        virtual
-        returns (
-            uint256 _profit,
-            uint256 _loss,
-            uint256 _debtPayment
-        );
+     * Liquidate as many assets as possible to `want`, irregardless of slippage,
+     * up to `_amountNeeded`. Any excess will be re-invested with `adjustPosition()`.
+     * This function should return any realized losses incurred, and should also
+     * return the amount of `want` tokens made available by the liquidation.
+     * This function is used during emergency exit instead of `prepareReturn()` to
+     * liquidate all of the Strategy's positions back to the Vault.
```

Comment:
liquidate `_amountNeeded` of ...

Reply:
Is this clearer?
https://github.com/iearn-finance/yearn-vaults/pull/140/files#r549917601

Reply:
yep!